<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="selectEquation">
    <property name="geometry">
     <rect>
      <x>419</x>
      <y>20</y>
      <width>291</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите уравнение состояния</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectProblem">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Укажите какую задачу хотите решить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputParam1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="parametr1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Величина p, в Па</string>
    </property>
   </widget>
   <widget class="QLabel" name="parametr2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Величина T, в К</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputParam2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mu">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Количество вщества, в молях</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputMu">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="equations">
     <item>
      <widget class="QRadioButton" name="IdealGase">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Идеальный газ</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="vanDerVaals">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Уравнение Ван-дер-Ваальса</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="diterichi">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Уравнение Дитеричи</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="problemStatement">
     <item>
      <widget class="QRadioButton" name="pT">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Известны p и T, найти V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="pV">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Известны p и V, найти Т</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="VT">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Известны V и T, найти p</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Химический калькулятор</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
